test/client/spec/victory-primitives/bar.spec.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { shallow } from "enzyme";
+import Bar from "src/victory-primitives/bar";
+import SvgTestHelper from "../svg-test-helper";
+
+describe("victory-primitives/bar", () => {
+  const baseProps = {
+    data: [
+      {_x: 2, x: 2, _y: 4, y: 4, eventKey: 0},
+      {_x: 3, x: 3, _y: 5, y: 5, eventKey: 1}
+    ],
+    datum: {_x: 2, x: 2, _y: 4, y: 4, eventKey: 0},
+    x: 2,
+    y: 10,
+    y0: 0,
+    width: 5,
+    padding: 2
+  };
+
+  it("should render a vertical bar", () => {
+    const wrapper = shallow(<Bar {...baseProps}/>);
+    const barShape = SvgTestHelper.getBarShape(wrapper);
+
+    expect(barShape.height).to.eql(10);
+  });
+
+  it("should render a horizontal bar", () => {
+    const props = Object.assign({}, baseProps, {horizontal: true});
+
+    const wrapper = shallow(<Bar {...props}/>);
+    const barShape = SvgTestHelper.getBarShape(wrapper);
+
+    expect(barShape.width).to.eql(10);
+  });
+
+  it("should apply a default width", () => {
+    const wrapper = shallow(<Bar {...baseProps}/>);
+    const barShape = SvgTestHelper.getBarShape(wrapper);
+
+    expect(barShape.width).to.eql(0.5);
```

Comment:
This default width seems arbitrary. Can you include an explanation or calculation of the default width (from padding and data length)